live-examples/css-examples/basic-box-model/width.css

```diff
@@ -0,0 +1,6 @@
+#example-element {
+    background-color: #E4F0F5;
```

Comment:
Style nit: use lowercase for the letters: `#e4f0f5`.

@schalkneethling , is there a style guide we can link from the contributing docs? It seems unfair to complain about something we don't document.